### PR TITLE
Closes #117

### LINKED ISSUE #117: Create dataset loader for GNormPlus corpus (BC2GM+BC2GN)
- Task: NER, NEN
- License: ? ("freely available from website")
- Format: BioC/PubTattor
- Language: English
- Citation:
Wei C-H, Kao H-Y, Lu Z. GNormPlus: An Integrative Approach for Tagging Gene, Gene Family and Protein Domain. BioMed Research International Journal, Text Mining for Translational Bioinformatics special issue, BioMed Research International Journal, Article ID 918710; DOI: dx.doi.org/10.1155/2015/918710 (2015)

- Source: 
- - https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/gnormplus/
- - https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/tmTools/download/GNormPlus/GNormPlusCorpus.zip

### PR DESCRIPTION
**Pull request by mart1nro**
### Checkbox

- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `biodatasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_BIGBIO_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `BigBioConfig` for the source schema and one for a bigbio schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_bigbio biodatasets/my_dataset/my_dataset.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: biodatasets/gnormplus/gnormplus.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                                "offsets": [span],
+                            }
+                            for passage, span in zip(document.passages, passage_spans)
+                        ],
+                        "entities": [
+                            {
+                                "id": next(uid),
+                                "type": entity.infons["type"],
+                                "text": [entity.text],
+                                "offsets": [(loc.offset, loc.end) for loc in entity.locations],
```
Comment (sg-wbi) [inline biodatasets/gnormplus/gnormplus.py:217]: Can you please modify this to use `utils.parsing.get_text_and_offsets_from_bioc_ann`?

**Review by sg-wbi — COMMENTED** [review]
@mart1nro Thank you for working on this! There are only two minor things to adjust:
1: see my comment
2: some of the entities have multiple normalizations, with `db_id`s joined with a `,` or a `;`. Can you please split them up [like here](https://github.com/bigscience-workshop/biomedical/blob/master/biodatasets/bc5cdr/bc5cdr.py#L222)

**Comment by mart1nro** [comment]
@sg-wbi 
I split the ids now. Two questions:

Some ids look like this:
`17494192(Tax:10376)`
Do I have to handle these differently?

Should I implement the ids splitting also for the source schema?

**Comment by sg-wbi** [comment]
> Some ids look like this:
17494192(Tax:10376)
Do I have to handle these differently?

I forgot about those. You need to parse them: `17494192` is the `db_id`.  In `source` I would add a `taxid` field containing `10376`, in `bigbio` schema we can throw this away, since the `taxid` can be retrieved from NCBI Gene.

> Should I implement the ids splitting also for the source schema?

Yes please

**Review by sg-wbi — APPROVED** [review]
Thank you for fixing this @mart1nro ! LGTM!

**Pull request merged**